client/package.json

```diff
@@ -32,13 +32,12 @@
     "react-copy-to-clipboard": "^5.0.2",
     "react-dom": "^16.13.1",
     "react-dropzone": "^10.2.2",
-    "react-helmet": "^6.1.0",
-    "react-popper": "^2.2.3",
-    "react-redux": "^7.2.1",
-    "react-router-dom": "^5.2.0",
-    "react-scroll-sync": "^0.8.0",
-    "react-svg-gauge": "^1.0.10",
-    "react-tippy": "^1.4.0",
+    "react-helmet": "^6.0.0",
+    "react-popper": "^1.3.7",
+    "react-redux": "^7.2.0",
+    "react-router-dom": "^5.1.2",
+    "react-scroll-sync": "^0.7.1",
+    "react-tippy": "^1.3.4",
```

Comment:
This has downgraded a bunch of our frontend packages.

I suggest locally resetting to cceda71155bb85daadf3b4f98f3d52f4daee20dc, then rebasing onto `release/5.0.0`. This will make for a much cleaner PR without the merge commits. I would copy your branch before attempting this.